### ISSUE TITLE
Can't specify LocalAI default model

### ISSUE THREAD

**Issue by 2xaronl** [nextcloud/integration_openai#47]
### Which version of integration_openai are you using?

1.0.13

### Which version of Nextcloud are you using?

27.1.1

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Firefox 117.0.1

### Describe the Bug

I can't see a way to specify the default model to be used in the Administration settings for my LocalAI connected account.  I get notifications that a task has failed if I, for instance, send a Free Prompt task of 'How are you?' to Nextcloud Assistant.

On the other hand, when I send a request from my Nextcloud server to my LocalAI server from the cli with the command:
`curl http://LOCALAI_IP:8080/v1/chat/completions -H "Content-Type: application/json" -d '{"model": "wizard-vicuna-13b-uncensored-superhot-8k.ggmlv3.q6_K.bin", "messages": [{"role": "user", "content": "How are you?"}]}'`

I get the response:
`{"object":"chat.completion","model":"wizard-vicuna-13b-uncensored-superhot-8k.ggmlv3.q6_K.bin","choices":[{"index":0,"finish_reason":"stop","message":{"role":"assistant","content":"\nI'm good, thanks for asking."}}],"usage":{"prompt_tokens":0,"completion_tokens":0,"total_tokens":0}`

### Expected Behavior

I would expect that there would be an Admin setting to configure the default model for LocalAI.  Once configured, I would expect the output of a Nextcloud Assistant request to be similar to that of a cli curl request.

### To Reproduce

Create a Nextcloud Assistant 'Free Prompt' task and wait for the task to timeout.

**Comment by octopusx** [comment]
Hey, I am seeing the same behaviour. I have tested this on Nextcloud 27.1.1 and 27.1.2, integration_openai 1.0.13, Firefox 115.3.0esr.

I had openAI token configured in the past and that worked at the beginning. Now I deployed my own copy of localai.io's container, I can talk to it and get prompts just like the original poster here. But all of the requests made via Nextcloud's web UI fail. I noticed this in the logs of my NC instance:

```
{"reqId":"hNC55OahaD8m4XG2FIZy","level":2,"time":"2023-10-15T13:04:15+00:00","remoteAddr":"10.42.3.1","user":"octopusx","app":"integration_openai","method":"POST","url":"/apps/integration_openai/completions","message":"OpenAI API error : Host violates local access rules","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0","version":"27.1.2.1","exception":{"Exception":"OCP\\Http\\Client\\LocalServerException","Message":"Host violates local access rules","Code":0,"Trace":[{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/PrepareBodyMiddleware.php","line":64,"function":"OC\\Http\\Client\\{closure}","class":"OC\\Http\\Client\\DnsPinMiddleware","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/Middleware.php","line":31,"function":"__invoke","class":"GuzzleHttp\\PrepareBodyMiddleware","type":"->","args":["*** sensitive parameters replaced ***","*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/RedirectMiddleware.php","line":71,"function":"GuzzleHttp\\{closure}","class":"GuzzleHttp\\Middleware","type":"::","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/Middleware.php","line":63,"function":"__invoke","class":"GuzzleHttp\\RedirectMiddleware","type":"->","args":["*** sensitive parameters replaced ***","*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/HandlerStack.php","line":75,"function":"GuzzleHttp\\{closure}","class":"GuzzleHttp\\Middleware","type":"::","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/Client.php","line":331,"function":"__invoke","class":"GuzzleHttp\\HandlerStack","type":"->","args":["*** sensitive parameters replaced ***","*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/Client.php","line":168,"function":"transfer","class":"GuzzleHttp\\Client","type":"->","args":["*** sensitive parameters replaced ***","*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/guzzlehttp/guzzle/src/Client.php","line":187,"function":"requestAsync","class":"GuzzleHttp\\Client","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/Http/Client/Client.php","line":301,"function":"request","class":"GuzzleHttp\\Client","type":"->","args":["*** sensitive parameters replaced ***","https://local-ai.octopusx.de/v1/chat/completions",["/var/www/html/data/files_external/rootcerts.crt",120,[["Closure"]],["Nextcloud OpenAI integration","application/json","gzip"],"{\"model\":\"gpt-3.5-turbo\",\"messages\":[{\"role\":\"user\",\"content\":\"What is a porcupine?\"}],\"max_tokens\":1000,\"n\":1,\"user\":\"octopusx\"}",[false],true]]},{"file":"/var/www/html/custom_apps/integration_openai/lib/Service/OpenAiAPIService.php","line":357,"function":"post","class":"OC\\Http\\Client\\Client","type":"->","args":["https://local-ai.octopusx.de/v1/chat/completions",[120,["Nextcloud OpenAI integration","application/json"],"{\"model\":\"gpt-3.5-turbo\",\"messages\":[{\"role\":\"user\",\"content\":\"What is a porcupine?\"}],\"max_tokens\":1000,\"n\":1,\"user\":\"octopusx\"}"]]},{"file":"/var/www/html/custom_apps/integration_openai/lib/Service/OpenAiAPIService.php","line":139,"function":"request","class":"OCA\\OpenAi\\Service\\OpenAiAPIService","type":"->","args":["*** sensitive parameters replaced ***","chat/completions",["*** sensitive parameters replaced ***",[["user","*** sensitive parameters replaced ***"]],"*** sensitive parameters replaced ***","*** sensitive parameters replaced ***","*** sensitive parameters replaced ***"],"POST"]},{"file":"/var/www/html/custom_apps/integration_openai/lib/Controller/OpenAiAPIController.php","line":83,"function":"createChatCompletion","class":"OCA\\OpenAi\\Service\\OpenAiAPIService","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"createCompletion","class":"OCA\\OpenAi\\Controller\\OpenAiAPIController","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\OpenAi\\Controller\\OpenAiAPIController"],"createCompletion"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":183,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\OpenAi\\Controller\\OpenAiAPIController"],"createCompletion"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\OpenAi\\Controller\\OpenAiAPIController","createCompletion",["OC\\AppFramework\\DependencyInjection\\DIContainer"],["integration_openai.openAiAPI.createCompletion"]]},{"file":"/var/www/html/lib/base.php","line":1068,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/integration_openai/completions"]},{"file":"/var/www/html/index.php","line":36,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/lib/private/Http/Client/DnsPinMiddleware.php","Line":143,"message":"OpenAI API error : Host violates local access rules","exception":[],"CustomMessage":"OpenAI API error : Host violates local access rules"},"id":"652be3a2a2728"}
```

This leads me to believe that there is something wrong with the `connected accounts` admin config form perhaps? It seems it is disregarding that I am changing the config to be pointing to a local instance instead of OpenAI's public API...

**Comment by MB-Finski** [comment]
@octopusx Your error seems to refer to a config error: Your error may be resolved by setting 'allow_local_remote_servers'=>true in your NC config. In any case, the integration is trying to connect to https://local-ai.octopusx.de/v1/chat/completions and not openAI.

**Comment by octopusx** [comment]
@MB-Finski Well spotted, thanks for the suggestion. That fixed it for me. I wonder if it's worth mentioning this somewhere on the page where you enable the integration inside NC, if this setting is off by default won't that mean more people would stumble across this? Or is it already mentioned somewhere and I am just blind?

**Comment by MB-Finski** [comment]
The docs are in need of some love and attention, for sure! I'll see if I can arrange some time to update them.

**Comment by jkellerer** [comment]
Just for your info, when #69 is released, `allow_local_remote_servers` is no longer required for localai

**Comment by MB-Finski** [comment]
Fixed in v1.2.0
